### PR TITLE
inventory/vultr: Adressing few comments

### PR DESCRIPTION
**Pull request by Spredzy**
##### SUMMARY

inventory/vultr: Adressing few comments

##### ISSUE TYPE

 - Bugfix Pull Request

##### COMPONENT NAME

  - inventory/vultr

##### ANSIBLE VERSION

  - devel

##### ADDITIONAL INFORMATION

  - N/A

### CODE REVIEW

File: lib/ansible/plugins/inventory/vultr.py [hunk 1]
```diff
@@ -116,8 +116,8 @@ def _retrieve_servers(api_key):
         if not servers_list:
             return []
 
-        return [server for id, server in servers_list.items()]
-    except ValueError as e:
+        return [servers_list[server] for server in servers_list.keys()]
```
Comment (pilou-) [inline lib/ansible/plugins/inventory/vultr.py]: my previous comment was unclear (s/keys/values/): could not that be `return servers_list.values()` ?

**Review by pilou- — COMMENTED** [review]
Should `ansible_host` defined when `hostname` is name ?

**Comment by Spredzy** [comment]
@pilou- last comment has been adressed in latest patchset

**Review by pilou- — APPROVED** [review]
shipit

**Review by resmo — APPROVED** [review]
verified. LGTM

shipit

**Pull request merged**